api/src/main/java/org/infinispan/api/reactive/Caches.java

```diff
@@ -0,0 +1,28 @@
+package org.infinispan.api.reactive;
+
+import org.infinispan.api.configuration.CacheConfiguration;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 13.0
+ **/
+public interface Caches {
+   <K, V> Uni<Cache<K, V>> create(String name, CacheConfiguration cacheConfiguration);
+
+   <K, V> Uni<Cache<K, V>> create(String name, String template);
+
+   <K, V> Uni<Cache<K, V>> cache(String name);
```

Comment:
Does this just retrieve an existing cache?

IMO `caches().get("mycache")` is clearer than `caches().cache("mycache")`

Reply:
Yes, since we are using verbs for create and remove, that's better

Reply:
Done